Assets/Scripts/Objects/GameResource.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public enum GameResourceType
+{
+    food,
+    oxygen,
+    water,
+    elecricity,
+
+    metals,
+    rareMetals,
+    dust,
+    radioactive,
+
+    concrete,
+    electronics,
+    machineParts,
+    steel,
```

Comment:
```suggestion
    Food,
    Oxygen,
    Water,
    Elecricity,

    Metals,
    RareMetals,
    Dust,
    Radioactive,

    Concrete,
    Electronics,
    MachineParts,
    Steel,
```

Should be better to use Pascal Casing for names of enumeration types, as in [ .NET Design-Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations)